### PR TITLE
Added the Trademark Symbol in md files

### PR DESCRIPTION
**Pull request by KrithikaGanesan**
Added the Trademark Symbol in md files

### CODE REVIEW

File: xamarin-android/ProgressBar/Overview.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
 layout: post
-title: Overview | Xamarin.Android | Syncfusion
-description: overview
-platform: Xamarin.Android
+title: Overview | Xamarin.Android | Syncfusion&reg;
+description: Syncfusion&reg; ProgressBar (SfProgressBar) control overview and the key features are available in Xamarin.Android
```
Comment (RajaVigneshB) [inline xamarin-android/ProgressBar/Overview.md:4]: Description needs some improvement. refer the below files and make changes as per it.

File: xamarin-android/SfDiagram/OverviewPanel.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Overview for Essential Xamarin.Android Diagram | Syncfusion
-description: Describes how to add overview panel and its customizion in diagram for Xamarin.Android
-platform: Xamarin.Android
+title: Overview for Essential&reg; Xamarin.Android Diagram | Syncfusion&reg;
+description: Describes how to add overview panel and its customizion in Syncfusion&reg; Diagram control for Xamarin.Android
```
Comment (RajaVigneshB) [inline xamarin-android/SfDiagram/OverviewPanel.md:3]: Instead of `control` use `(SfDiagram)`

**Review by RajaVigneshB — CHANGES_REQUESTED** [review]

File: xamarin-android/ProgressBar/Overview.md (hunk 1 above)
Comment (KrithikaGanesan) [reply xamarin-android/ProgressBar/Overview.md]: I have updated the description

File: xamarin-android/SfDiagram/OverviewPanel.md (hunk 2 above)
Comment (KrithikaGanesan) [reply xamarin-android/SfDiagram/OverviewPanel.md]: I have modified

**Review by RajaVigneshB — APPROVED** [review]

**Pull request merged**